config/locales/models/type_de_champ/fr.yml

```
@@ -58,6 +58,19 @@ fr:
           cojo: "Accréditation Paris 2024"
           expression_reguliere: 'Expression régulière'
           referentiel: 'Référentiel à configurer (avancé)'
+        default_libelle:
+          annuaire_education: Établissement
+          communes: Commune
+          departements: Département
+          dossier_link: Numéro de dossier déposé sur Démarches Simplifiées
```

```suggestion
dossier_link: Numéro de dossier déposé sur %{app_name}
```

et passer à l'i18n `app_name: APPLICATION_NAME`.
Le nom est dynamique car chaque instance a son nom.